qfs/chain.json

```diff
@@ -0,0 +1,77 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "qfs",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://qfs.com/",
+    "pretty_name": "qfs",
```

Comment:
Why not call it Quantum Financial Systems?

Reply:
Okay I will change the "pretty_name" as Quantum Financial Systems .
@JeremyParish69

Reply:
I don't see that you've made this change yet